src/appsco-customers-page.html

```diff
@@ -330,206 +237,6 @@
-            },
-
-            _onPageLoaded: function() {
-                this.pageLoaded = true;
-                this.fire('page-loaded');
-            },
-
-            _onCustomerSelected: function(event) {
-                var customer = event.detail.customer;
-
-                clearTimeout(this._customerAction);
-
-                this._customerAction = setTimeout(function() {
-
-                    if (customer.selected) {
-                        this._onViewInfo(event);
-                    }
-                    else {
-                        var selectedCustomer = this.$.appscoCustomers.getFirstSelectedCustomer();
-
-                        if (!selectedCustomer) {
-                            this.hideInfo();
-                            this._setDefaultCustomer();
-                        }
-                        else if (customer.self === this.customer.self) {
-                            this._handleInfo(selectedCustomer);
-                        }
-                    }
-
-                }.bind(this), 10);
-            },
-
-            _onCustomerInfoEdit: function() {
-                this.fire('info-edit-customer', {
-                    customer: this.customer
-                });
-            },
-
-            _showInfo: function() {
-                if (this._customerExists) {
-                    this.$.appscoContent.showSection('info');
-                    this._infoShown = true;
-                }
-            },
-
-            _handleInfo: function(customer) {
-                this.set('customer', customer);
-
-                if (!this._infoShown) {
-                    this._showInfo();
-                }
-            },
-
-            _onViewInfo: function(event) {
-                this._handleInfo(event.detail.customer);
-            },
-
-            _removeCustomerRoles: function(roles) {
-                this.$.appscoCompanyCustomerRoles.removeCustomerRoles(roles);
-            },
-
-            _removeCustomerContacts: function(contacts) {
-                this.$.appscoCompanyCustomerContacts.removeCustomerContacts(contacts);
-            },
-
-            _removeCustomerResources: function(resources) {
-                this.$.appscoCompanyCustomerResources.removeCustomerResources(resources);
-            },
-
-            _removeCustomerItems: function(items, resourceType) {
-
-                switch (resourceType) {
-                    case 'resource':
-                        this._removeCustomerResources(items);
-                        break;
-
-                    case 'account':
-                        this._removeCustomerRoles(items);
-                        break;
-
-                    case 'contact':
-                        this._removeCustomerContacts(items);
-                        break;
-
-                    default:
-                        return false;
-                }
-            },
-
             _onPageAnimationFinished: function(event) {
```

Comment:
This is no needed, I think. Check if this is called from some point inside component.